mne_bids/dig.py

```diff
@@ -269,11 +271,20 @@ def _coordsystem_json(*, raw, unit, hpi_coord_system, sensor_coord_system,
                          .format(coord_frame))
 
     # get the coordinate frame description
+    try:
+        sensor_coord_system, sensor_coord_system_mne = sensor_coord_system
+    except ValueError:  # pragma: no cover
+        pass
     sensor_coord_system_descr = (COORD_FRAME_DESCRIPTIONS
                                  .get(sensor_coord_system.lower(), "n/a"))
-    if sensor_coord_system == 'Other' and verbose:
-        print('Using the `Other` keyword for the CoordinateSystem field. '
-              'Please specify the CoordinateSystemDescription field manually.')
+    if sensor_coord_system == 'Other':
+        if verbose:
+            print('Using the `Other` keyword for the CoordinateSystem field. '
+                  'Please specify the CoordinateSystemDescription field '
+                  'manually.')
+        sensor_coord_system_descr = (COORD_FRAME_DESCRIPTIONS
+                                     .get(sensor_coord_system_mne.lower(),
```

Comment:
It is possible that `sensor_coord_system_mne` is not available?

Reply:
I don't know, but it's a good point --> I made this more robust in https://github.com/mne-tools/mne-bids/pull/706/commits/e734cefdb4815fc4033263e3ec4a55c941b6cf72